_episodes/05-creating-container-images.md

```diff
@@ -117,8 +117,22 @@ Let's break this file down:
 - The first line, `FROM`, indicates which container we're starting with.
 - The next two lines `RUN`, will indicate installation commands we want to run. These
 are the same commands that we used interactively above.
-- The last line, `CMD` indicates the default command we want the container to run,
-if no other command is provided.
+- The last line, `CMD`, indicates the default command we want the
+container to run, if no other command is provided. It is recommended
+to provide `CMD` in *exec-form*, that is a JSON-array which contains
+as first element the executable, optionally followed by its
+arguments. For example: `CMD ["ls", "-lF", "--color", "/etc"]`
+
+> ## *shell-form* and *exec-form* for CMD
+> Another way to specify the `CMD` is the *shell-form*. Her you just
+> type the command as you would call it from the command line. Docker
+> then silently runs this command in the image's standard shell.  `CMD
+> cat /etc/passwd` is equivalent to `CMD
+> ["/bin/sh", "-c", "cat /etc/passwd"]`. We recommend to prefer the
+> more explicit *exec-form* to avoid hard to interpret behavior in
```

Comment:
I'd state this in a postive way -- not what we are avoiding but what we gain. So, we recommend "because we will be able to create more flexible container command options and make sure complex commands are unambiguous in thsi format."